ElasticFrameProtocol.h

```diff
@@ -75,8 +73,9 @@ extern "C" {
 #define PRIORITY_P3     0b01100000 // God-mode priority (not implemented)
 #define UNDEFINED_FLAG  0b10000000 // TBD
 
-#define EFP_MAJOR_VERSION 0
-#define EFP_MINOR_VERSION 4
+constexpr uint8_t EFP_MAJOR_VERSION = 0;
+constexpr uint8_t EFP_MINOR_VERSION = 4;
```

Comment:
Should we increase version, just for the record? After all, you have made some changes, most of them (hopefully) not noticeable, except for the removed loggings.

Reply:
This PR should not have changed the behavior of the repo, not even fixed any bugs, so I would argue this would not bump the version number. This PR ought to just be a refactorization of the code style.
And the logs should not have been used for anything except manual debugging, so no need to bump the version because of them, if you ask me.